index.js

```diff
@@ -447,12 +443,7 @@ exports.resolveFunctions = function resolveFunctions (funcSet, dir = '/wasm/') {
     try {
       const wast = fs.readFileSync(wastPath)
       funcs.push(wast.toString())
-    } catch (e) {
-      // FIXME: remove this once every opcode is implemented
-      //        (though it should not cause any issues)
-      console.error('Inserting MISSING opcode', func)
-      funcs.push(`(func $${func} (param $sp i32) (result i32) (unreachable))`)
-    }
+    } catch (e) {}
```

Comment:
JUMP, JUMPDEST, PUSH, POP are not file so it throws there. Ill see if there is a nice way to remove them from the map